_includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md

```diff
@@ -2,7 +2,7 @@
 
 The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Vehicle-sharing App](movr.html).
 
-To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the [Geo-Partitioned Replicas Topology](topology-geo-partitioned-replicas.html) applied to the `movr` database.
+To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the `movr` database.
```

Comment:
Interesting - was this a redirect?  I looked at `v21.1/topology-geo-partitioned-replicas.md` and it looks like a normal docs page

Reply:
Yes it is. I'm not sure why this redirect isn't working, but `/stable` is.

I'm also now discovering that the redirect just straight up does not work and produces a 404 if navigated to directly, so I think I need to tweak my redirect logic.

Reply:
Redirects fixed.